std_msgs/Header header

geometry_msgs/Point32 center3d

float32 example

string content
